downstream/modules/automation-hub/con-remote-repos.adoc

```diff
@@ -2,7 +2,7 @@
 
 = About remote repositories
 
-You can configure your local {HubName} to sync with Red Hat Certified Collections hosted in your organization repository on console.redhat.com and to your choice of collections in {Galaxy} by configuring remote repositories.
+By configuring remote repositories, you can set your {PrivateHubName} to synchronise Red Hat Certified Collections hosted in your organization's repository on `{Console}` and your choice of collections in {Galaxy}.
 
-Each remote repository located in menu:Repo Management[Remote] provides information for both *community* and *rh-certified* about when the repo was *last updated* and when content was *last synced*. 
-You can add new content to {HubName} at any time using the *Edit* and *Sync* features included on the menu:Repo Management[Remote] page.
+Each remote repository located in menu:Repo Management[Remote] provides information for both the *community* and *rh-certified* repository about when the reposiitory was *last updated* and when content was *last synced*. 
```

Comment:
```suggestion
Each remote repository located in menu:Repo Management[Remote] provides information for both the *community* and *rh-certified* repository about when the repository was *last updated* and when content was *last synced*. 
```

Also, can we use "synced"?  It's obvious what it means, but it might cause problems for translation.